spidermon/contrib/scrapy/monitors.py

```diff
@@ -288,6 +288,24 @@ class UnwantedHTTPCodesMonitor(BaseScrapyMonitor):
             500: 0,
         }
 
+    Furthermore, instead of being a numeric value, the code accepts a dictionary which can
+    contain any of two keys: ``max_count`` and ``percentage``. The former refers to an
+    absolute value and works the same way as setting an integer value. The latter refers
+    to a percentage of the total number of requests the spider made. If both are set, the
+    monitor will pick the smallest of the two values. If none are set, it will default to
```

Comment:
> If both are set, the monitor will pick the smallest of the two values.

I think this could lead to confusion. e.g. to think that, in the case of `{"max_count": 100, "percentage": 0.5}`, `max_count` is always ignored because `0.5 < 100`.

What about something in the lines of:

> If both are set, the monitor will fail if any of the conditions are met.

Which is what the implementation achieves, even if it is through the use of `min(max_count or requests, percentage*requests)`.

Reply:
Makes sense! Will update the text.